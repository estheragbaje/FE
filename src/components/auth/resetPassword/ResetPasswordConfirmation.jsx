import React from 'react';
import { Link } from 'react-router-dom';
import ResetPasswordDiv from './StyledResetPassword';

function ResetPasswordConfirmation() {
  return (
    <ResetPasswordDiv>
      <div className="confirmContainer text-center border border-light p-5 z-depth-1">
        <p className="h4 mb-4">Password Reseted</p>
        <p className="mb-4">Check your email</p>

        <Link to="/">
          <u>Return to home page</u>
        </Link>
      </div>
    </ResetPasswordDiv>
  );
}

export default ResetPasswordConfirmation;
