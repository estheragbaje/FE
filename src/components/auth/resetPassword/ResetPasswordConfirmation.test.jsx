import React from 'react';
import { shallow } from 'enzyme';

import ResetPasswordConfirmation from './ResetPasswordConfirmation';

describe('<ResetPasswordConfirmation />', () => {
  it('renders the ResetPasswordConfirmation component correctly', () => {
    const wrapper = shallow(<ResetPasswordConfirmation />);
    expect(wrapper).toMatchSnapshot();
  });
});
