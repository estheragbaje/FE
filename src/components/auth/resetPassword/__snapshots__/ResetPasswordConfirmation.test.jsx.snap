// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResetPasswordConfirmation /> renders the ResetPasswordConfirmation component correctly 1`] = `ShallowWrapper {}`;
